.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: Check PR title
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: deepakputhraya/action-pr-title@master
+      with:
+        regex: '^(PUB-[0-9]{3} [a-zA-Z0-9._\- ]+)|(\[Snyk\].+)$' # Regex the title should match.
+        allowed_prefixes: 'PUB-,[Snyk]' # title should start with the given prefix
+        prefix_case_sensitive: true # title prefix are case insensitive:  
```

Comment:
so here i think we can add (under the steps catagory)
```
- name: Run linting
  run: yarn lint
```
to run the linting 
and under the jobs catagory something like 
```
- name: CI Checks
  run: yarn cichecks

Reply:
This was added using the rules for protecting the branch:
![image](https://user-images.githubusercontent.com/80695759/142999411-01b8294d-24d1-4ef7-a0b3-5876fd342da3.png)
No need to include these lines of code